docs/commands/preview-docs.md

```diff
@@ -1,39 +1,98 @@
 # `preview-docs`
 
-Preview the API reference docs on your local machine.
+## Introduction
 
-If you have a license key, you will have a preview of the premium Redocly API reference docs. The [`login`](./login.md) command also generates a preview of the premium Redocly API reference docs.
+With this command, you can preview the API reference docs on your local machine.
 
-Otherwise, you'll get a preview of Redoc community edition.
+If you have a license key, you will get a preview of the premium Redocly API reference docs. If you don't, you will get a preview of Redoc community edition.
```

Comment:
suggestion: We could link to the Reference docs overview page and the Redoc quickstart guide here to connect the products and help readers access the info about each product quickly.